### PR TITLE
Untested Auto Selection Code (from vision)

### PR DESCRIPTION
**Pull request by BradyBangasser**
See RobotContainer.java

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -63,11 +60,27 @@ public RobotContainer() {
...
     NamedCommands.registerCommand("intake_command", new IntakeCommand(m_flinger, m_intake));
+
+    autoChooser = AutoBuilder.buildAutoChooser();
+
+    SmartDashboard.putData("Auto Chooser", autoChooser);
   }
 
+  /**
+   * Get the selected Auto Command 
+   * @implNote We should write the robot code in C++
```
Comment (KeithRieck) [inline src/main/java/frc/robot/RobotContainer.java:71]: Hmm...

**Review by KeithRieck — APPROVED** [review]
Looks fine, and straight out of the PathPlanner docs.

We should test it on the robot, of course.

**Review by HenryEwald — DISMISSED** [review]
I like the code here but would it be possible to have the vision set what's selected on the autoChooser? getAutonomousCommand() is only called when auto starts so the system may only use a vision auto when the qr code is being held up to the camera right as auto starts, setting the selected auto also allows drivers to make last second changes and might let the selected auto permeate between rounds.

**Comment by beardedone55** [comment]
Overall, this looks good, Bardy.

**Comment by BradyBangasser** [comment]
> I like the code here but would it be possible to have the vision set what's selected on the autoChooser? getAutonomousCommand() is only called when auto starts so the system may only use a vision auto when the qr code is being held up to the camera right as auto starts, setting the selected auto also allows drivers to make last second changes and might let the selected auto permeate between rounds.

I took a quick look at the docs for the SendableChooser Docs and there doesn't seem to be a good way of doing this. The only way I can think of is if we assign it as the default value, which should be done during the initiation phase and changing it multiple times seems like it would be a bad practice and we would also need to use files to persist that data between matches which we could do, but it would be interesting to integrate with the Path Planner AutoSelector. I think these would be fun to implement but I doubt we would have time to do that before comp

**Comment by beardedone55** [comment]
> > I like the code here but would it be possible to have the vision set what's selected on the autoChooser? getAutonomousCommand() is only called when auto starts so the system may only use a vision auto when the qr code is being held up to the camera right as auto starts, setting the selected auto also allows drivers to make last second changes and might let the selected auto permeate between rounds.
> 
> I took a quick look at the docs for the SendableChooser Docs and there doesn't seem to be a good way of doing this. The only way I can think of is if we assign it as the default value, which should be done during the initiation phase and changing it multiple times seems like it would be a bad practice and we would also need to use files to persist that data between matches which we could do, but it would be interesting to integrate with the Path Planner AutoSelector. I think these would be fun to implement but I doubt we would have time to do that before comp

Brady, can't your vision code just always send the string associated with the last QR code it saw?

**Review by KeithRieck — APPROVED** [review]

**Review by beardedone55 — APPROVED** [review]
Looks good.

**Pull request merged**